docs/content/templates/home/Home.tsx

```diff
@@ -13,17 +14,20 @@ import {
 } from '@ag.ds-next/react/hero-banner';
 import { Text } from '@ag.ds-next/react/text';
 
+const placeholderBanner = 'placeholder/hero-banner.jpeg';
+const placeholderImage = 'placeholder/600x260.png';
+
 export function Home() {
 	return (
-		<>
+		<Box
+			flexGrow={1}
+			as="main"
+			id="main-content"
+			tabIndex={-1}
+			css={{ '&:focus': { outline: 'none' } }}
+		>
```

Comment:
Should this block be in the `SiteLayout` OR should we make sure we include it in every template, wrapping the "main-content". 

Probably the latter right?